### ISSUE TITLE
Aria-required property is not defined for 'Autoscale' and 'Manual' radio controls under 'New collection' blade.

### ISSUE THREAD

**Issue by ato9000** [Azure/cosmos-explorer#1016]
Note: User credentials should NOT be included in the bug. 
Repro Steps:
Open URL: https://ms.portal.azure.com/ and login using valid credentials.
Navigate to "Azure Cosmos DB " from left pane and select it.
Navigate to any of the Azure Cosmos DB which is already created and select it.
Navigate to "Add Collection" Menu item/Button and select it.
Navigate through all the controls under add Collection/New collection blade.
Navigate to 'Autoscale' and 'Manual' radio controls present under the blade.
Press F12 key to open the Developers Tool and inspect the code.
Verify the issue.

Actual Result:
Aria-required property is not defined for 'Autoscale' and 'Manual' radio controls under 'New collection' blade. Due to which screen reader is not reading "required" information for radio controls.

Expected Result:
Aria-required property should be defined for 'Autoscale' and 'Manual' radio controls under 'New collection' blade so that the screen reader announce required information.

Note:
Similar issue exists under 'New Database' blade.
This issue also exists for similar control present under 'New Graph' blade of Data explorer screen for Created cosmos DB  after selecting "Gremlin(graph)" option.
This issue exists throughout the application for similar controls.

![image](https://user-images.githubusercontent.com/8519407/132721534-207fb357-f7f6-4a87-ac20-08ec5dec643b.png)

**Comment by ato9000** [comment]
https://msdata.visualstudio.com/CosmosDB/_workitems/edit/1216083
